client/src/views/AboutUsView.vue

```diff
@@ -41,9 +41,13 @@ export default defineComponent({
   color: $white;
 }
 
+.h1-header {
+  text-align: start;
+}
+
```

Comment:
If you merge main, do you still need this? I removed centered text out of h1-header into its own mixin

Reply:
Nope - thanks!